vllm_ascend/worker/model_runner.py

```diff
@@ -368,6 +376,26 @@ def __post_init__(self):
             self.lora_index_mapping = []
             self.lora_prompt_mapping = []
 
+        def __repr__(self) -> str:
```

Comment:
qq: what's this func used for? I assume it's not related to this change but better to have, right?